src/storage/src/hummock/sstable/block.rs

```diff
@@ -48,7 +48,18 @@ impl Block {
         let buf = match compression {
             CompressionAlgorithm::None => buf.slice(..buf.len() - 9),
             CompressionAlgorithm::Lz4 => {
-                let mut decoder = Decoder::new(buf.reader())
+                let mut decoder = lz4::Decoder::new(buf.reader())
```

Comment:
I think we should also ignore the last 9 bytes for lz4?

Reply:
Maybe we can define a new variable

let compressed_data = &buf[..buf.len()-9] ahead, and use it in the following program.